lib/rucio/web/rest/webpy/v1/did.py

```diff
@@ -73,6 +75,7 @@
     '/new', 'NewDIDs',
     '/resurrect', 'Resurrect',
     '/list_dids_by_meta', 'ListByMeta',
+    '%s/follow/(.*)' % SCOPE_NAME_REGEXP, 'Follow',
```

Comment:
only `%s/follow' 
There shouldn't be another parameter in the REST. the account should be part of the data